### PR TITLE
Feature/profile details

### PR DESCRIPTION
**Pull request by amar-josh**
Profile Details Screen UI

### CODE REVIEW

File: src/app/screens/PeerlyScreens/Components/RatingBar.tsx [hunk 1]
```diff
@@ -28,33 +30,37 @@ const RatingBar: React.FC<RatingBarProps> = ({
...
+          <View style={styles.sliderContainer}>
+            <Slider
+              disabled={disableSlider || isRewardAlreadyGiven}
+              style={styles.slider}
+              minimumValue={0}
+              maximumValue={5}
+              step={1}
+              value={reward}
+              onValueChange={value => setReward(value)}
+              thumbImage={require('../../../../assets/images/starIcon.png')}
```
Comment (Chetan-Satpute-Josh) [inline src/app/screens/PeerlyScreens/Components/RatingBar.tsx:52]: Can we please move this to icons file, like we did for other icons?

File: src/app/screens/PeerlyScreens/HomeScreen/index.tsx [hunk 2]
```diff
@@ -97,26 +100,33 @@ const HomeScreen = ({navigation}) => {
...
-              profileDetails?.profile_image_url
-                ? {uri: profileDetails?.profile_image_url}
-                : require('../../../../assets/images/profile.png')
-            }
-            style={styles.userAvatar}
-          />
+      <Pressable
+        onPress={() =>
+          navigation.navigate(PROFILE_DETAILS, {
+            details: profileDetails,
```
Comment (Chetan-Satpute-Josh) [inline src/app/screens/PeerlyScreens/HomeScreen/index.tsx:106]: We should avoid passing data from one screen to another. 
We should pass an `id` and then fetch the details on the next screen itself.

**Review by Chetan-Satpute-Josh — CHANGES_REQUESTED** [review]

File: src/app/screens/PeerlyScreens/Components/RatingBar.tsx (hunk 1 above)
Comment (amar-josh) [reply src/app/screens/PeerlyScreens/Components/RatingBar.tsx:52]: Yes, I missed out it. I will add it and update.

File: src/app/screens/PeerlyScreens/HomeScreen/index.tsx (hunk 2 above)
Comment (amar-josh) [reply src/app/screens/PeerlyScreens/HomeScreen/index.tsx:106]: Sure, I will make this change and update.

Comment (amar-josh) [reply src/app/screens/PeerlyScreens/HomeScreen/index.tsx:106]: I have done the change in my upcoming PR so you will get updated code in next PR

**Pull request merged**